tests/test_download_epmc.py

```diff
@@ -0,0 +1,61 @@
+from unittest import mock
+import json
+import os
+
+import pytest
+
+from grants_tagger.download_epmc import download_epmc
+
+
+@pytest.fixture
+def download_path(tmp_path):
+    return tmp_path
+
+
+def create_month_data(month_path):
```

Comment:
Is this going to actually create data and delete it after the test?

Reply:
Or is it automatically deleted because of the fixture?

Reply:
automatically deleted because of the fixture